bool led_front
bool led_left
bool led_right
bool led_rear
int8 music_id
string screen_message
